cmd/ethermintd/config/config.go

```diff
@@ -57,27 +57,30 @@ func DefaultConfig() *Config {
 	}
 }
 
-// DefaultEVMConfig returns an EVM config with the JSON-RPC API enabled by default
-func DefaultEVMConfig() *EVMRPCConfig {
-	return &EVMRPCConfig{
-		Enable:     true,
-		RPCAddress: DefaultEVMAddress,
-		WsAddress:  DefaultEVMWSAddress,
-	}
-}
-
 // EVMRPCConfig defines configuration for the EVM RPC server.
 type EVMRPCConfig struct {
 	// RPCAddress defines the HTTP server to listen on
 	RPCAddress string `mapstructure:"address"`
 	// WsAddress defines the WebSocket server to listen on
 	WsAddress string `mapstructure:"ws-address"`
+	// Api defines the api namespaces that should be enabled
+	API string `mapstructure:"api"`
```

Comment:
I think this should be a slice
```suggestion
	API []string `mapstructure:"api"`
```

Reply:
Hey Fede! This is a string, we then split it to get a slice out of it at [line](https://github.com/tharsis/ethermint/pull/349/files#diff-4296685a7348b4ed68d276fc1a5f35779fb132578be1760454aeeb3ceb4f2c89R28)

Reply:
Oh nevermind, just saw there is a function that returns this as slice by default! Will commit the suggestions